### DISCUSSION TITLE
Keyboard tweaks

### DISCUSSION THREAD

**Discussion by schollz** [monome/norns#1610] [General]
The keyboard shortcuts in norns are wonderful - they are really intuitive, seamless and make navigation really fast. 

I am wondering whether there is room for adding a few tweaks the the shortcuts to improve the intuitivity (though I totally understand if my intuition is its own thing). Namely the following little tweaks:

- Allowing +/- keys to increase/decrease values in the parameter menu
- Allow holding up/down keys in the parameter menu to scroll through parameters instead of pressing each time
- Allow F1-F4 to automatically toggle the menu and jump into the menu, or if this is too intrusive then to simply allocate F5 as a menu toggle key

I'll also add one less minor tweak:

- Allowing the core keyboard to be triggered by osc messages with keyboard key and value (i.e. through `/remote/brd`)

The last one would make it easier to process keyboard commands from something other than the norns, to control the norns (i.e. a browser that captures keystrokes through javascript). 

All these tweaks I personally find helpful and have implemented in my own norns. If any of these are of interest, I can PR. But no big deal if none are of interest.

**Comment by tehn** [comment]
I'd be up for all of these.

**Reply by schollz** [reply]
👍 PR added
